unit/Makefile

```diff
@@ -16,14 +17,14 @@ LIBS += ../src/ansi-c/ansi-c$(LIBEXT) \
         ../src/cpp/cpp$(LIBEXT) \
         ../src/json/json$(LIBEXT) \
         ../src/linking/linking$(LIBEXT) \
-        ../src/util/util$(LIBEXT) \
-        ../src/big-int/big-int$(LIBEXT) \
-        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/pointer-analysis/pointer-analysis$(LIBEXT) \
         ../src/langapi/langapi$(LIBEXT) \
         ../src/assembler/assembler$(LIBEXT) \
         ../src/analyses/analyses$(LIBEXT) \
+        ../src/goto-programs/goto-programs$(LIBEXT) \
         ../src/solvers/solvers$(LIBEXT) \
+        ../src/util/util$(LIBEXT) \
+        ../src/big-int/big-int$(LIBEXT) \
```

Comment:
Ah OK, me neither! So the link command is:

```bash
clang++  -o unit_tests -Wl,--start-group unit_tests.o catch_example.o analyses/ai/ai_simplify_lhs.o -Wl,--end-group ../src/ansi-c/ansi-c.a ../src/cpp/cpp.a ../src/json/json.a ../src/linking/linking.a ../src/pointer-analysis/pointer-analysis.a ../src/langapi/langapi.a ../src/assembler/assembler.a ../src/analyses/analyses.a ../src/goto-programs/goto-programs.a ../src/solvers/solvers.a ../src/big-int/big-int.a ../src/util/util.a 
```

So all the libs somehow end up outside of the `--start-group`/`--end-group` flags, so I suppose somehow `LIBS` is not put inside that group, I'll investigate but don't know `src/common`.